### ISSUE TITLE
ACE Loadouts saved with Global Mobilisation RPG backpacks (both variants) don't save the backpack

### ISSUE THREAD

**Issue by Joe-Banks** [acemod/ACE3#10346]
**Mods/DLCs:**

- **Arma 3:** `2.16 Stable` 
- **CBA:** `3.17.1 Stable`
- **ACE3:** `3.17.1 Stable`
- **Global Mobilisation CDLC** `V1.5` 

**Description:**
Attempting to save a loadout with either GM RPG backpack causes the entry to "grey out", and the backpack is lost when the loadout is loaded. 

**Steps to reproduce:**
1. Go to the ACE 3 Arsenal (on a unit in Zeus/Editor, or the Virtual Arsenal)
2. Create a loadout with either `gm_gc_army_backpack_80_at_str` or `gm_gc_army_backpack_80_at_gry` in it
3. Save the loadout, then close the loadout menu
4. Go back to the loadout menu and attempt to load it
5. Observe that the RPG backpack will be missing

**Expected behavior:**
The expected outcome is that the RPG backpacks would save properly and load properly when the loadout is selected 

**Where did the issue occur?**
- Multiplayer Editor, Virtual Arsenal

**Log Files:**
- Was unsure if I needed to include this but popped here anyway - apologies for the several thousand lines of "Loadout is missing items" - I have a lot of loadouts across a lot of modpacks (Line 4469 looks like the right place?)
[ACE3_RPGIssue_ReportFile.txt](https://github.com/user-attachments/files/17171343/ACE3_RPGIssue_ReportFile.txt)

Relevant section of .rpt file:
"23:48:02 ACE_Arsenal - Loadout:
    Name: TEST
    Missing items: []
    Unavailable items: ["gm_backpack_base"]"

**Additional context:**
As mentioned above, the effected classnames are `gm_gc_army_backpack_80_at_str` and `gm_gc_army_backpack_80_at_gry`

Exported example loadout using the Strichtarn backpack (first one before save/loading, second one after loading);
`[[[],[],[],["gm_gc_army_uniform_soldier_80_str",[]],["gm_gc_army_vest_80_leader_str",[]],["gm_gc_army_backpack_80_at_str",[]],"","",[],["ItemMap","ItemGPS","","ItemCompass","ChemicalDetector_01_watch_F",""]],[["ace_earplugs",true]]]`

`[[[],[],[],["gm_gc_army_uniform_soldier_80_str",[]],["gm_gc_army_vest_80_leader_str",[]],[],"","",[],["ItemMap","ItemGPS","","ItemCompass","ChemicalDetector_01_watch_F",""]],[["ace_earplugs",true]]]`

**Comment by Joe-Banks** [comment]
Okay so, I used the command sent on Discord to test a variety of backpacks; 
-RPG Backpacks: Default Inventory of 3 rockets
-Flecktarn Backpack: No Inventory
-Medical Satchel: No Inventory
-MG Ammo Bag (Strichtarn): 2 75 round RPK mags
-Assault Pack, MG (Strichtarn): 4 75 round RPK mags

In testing this I also spotted the same thing happening with that MG Ammo Bag (`gm_gc_army_backpack_80_lmg_str`) and Assault Pack (`gm_gc_army_backpack_80_assaultpack_lmg_str`) which I hadn't noticed before. 

I tested saving a loadout with each Backpack in the CDLC, and it only happens with the two RPG bags previously identified, and the two Strichtarn MG bags just mentioned.

**Comment by Joe-Banks** [comment]
Yup, they're visible in the ACE Arsenal fine, it's only saving them that's the problem. How would the GM guys have to go about fixing this - making a second version of the backpack that's empty, and hiding the current preloaded one from the Arsenal?

**Comment by johnb432** [comment]
`gm_gc_army_backpack_80_at_gry` doesn't exist.

---

The best solution would be for GM to make classes available to the arsenal that don't have items. However, there are some areas where the arsenal doesn't handle such backpacks properly, so we *might* be able to fully address the problem on our end.

**Comment by johnb432** [comment]
> @johnb432 unsure if we go the roundabout way of fixing this (recreate the CBA loop and break if the base backpack's model isn't the same? Opens us up for more edge cases I think though. Alternatively break for scope 2 backpacks with inventory) or just mark as wontfix.

I'm not a big fan of either solution tbh, but the second one sounds more robust to me.

**Comment by Joe-Banks** [comment]
> `gm_gc_army_backpack_80_at_gry` doesn't exist.
> 
> The best solution would be for GM to make classes available to the arsenal that don't have items. However, there are some areas where the arsenal doesn't handle such backpacks properly, ~so we _might_ be able to fully address the problem on our end.~ EDIT: not possible as far as I can tell, GM needs to change stuff on their end.

Cool - I can always mention it over on GM's server and see what they think about it?

**Comment by Joe-Banks** [comment]
Update - They agree it should be an easy fix, looks like it should be sorted at some point in the future! Thanks for the help both

**Comment by TrakkarLP** [comment]
Hi!
I can even add more to it, after the Update yesterday, it isn't only the RPG backpacks, it's: Clothing, Wests, and more. In some cases even Ammo for the G3.
